src/routes/Chat/component.js

```diff
@@ -150,7 +150,7 @@ export default class Chat extends Component {
 			>
 				<Screen.Content nopadding>
 					{ incomingCallAlert && !!incomingCallAlert.show && <CallNotification { ...incomingCallAlert } dispatch={dispatch} />}
-					{ ongoingCall && ongoingCall.callStatus === 'accept' ? <CallIframe { ...incomingCallAlert } /> : null }
+					{ incomingCallAlert?.show && ongoingCall && ongoingCall.callStatus === 'accept' ? <CallIframe { ...incomingCallAlert } /> : null }
```

Comment:
I don't think we'd be needing to check this condition anymore right? .... Since `ongoingCall` property is not updated when call is expanded

Reply:
But what about when the call stop button is pressed?

Reply:
Okay that makes sense :+1: We'd still require this condition here